src/value.h

```diff
@@ -75,6 +77,8 @@ SIValue SI_DoubleVal(double d);
 SIValue SI_NullVal(void);
 SIValue SI_BoolVal(int b);
 SIValue SI_PtrVal(void* v);
+SIValue SI_NodeVal(void* v);
```

Comment:
SIValue SI_NodeVal(Node* n); ?
SIValue SI_EdgeVal(Edge* e); ?

Reply:
We get circular dependency issues if we actually include node.h and edge.h in this file. Right now, we've only had to include graph_entity.h in value.c, since all value-level comparisons are being performed by ID.